### PR TITLE
Change header links and auth buttons hover states

### PR DESCRIPTION
**Pull request by McCambley**
# Describe your changes 

Teeny PR. Changing the header links and auth form buttons hover states to match the figma. 

# I follow best practices 

- [x] My code is covered by unit tests 
- [x] This PR is less than [300 lines of code](https://www.atlassian.com/blog/git/written-unwritten-guide-pull-requests)
- [x] My code [smells good](https://en.wikipedia.org/wiki/Code_smell) 
- [x] I have tested my PR locally and proven it works. 
- [x] There is [no copy-paste](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

### CODE REVIEW

File: frontend/src/components/header/styledHeader.js [hunk 1]
```diff
@@ -20,9 +20,6 @@ export const Image = styled.img`
   margin-right: 16px;
   cursor: pointer;
   transition: opacity 0.3s ease;
```
Comment (rileydanejohnston) [inline frontend/src/components/header/styledHeader.js:22]: Do we still need this transition property if you removed the hover selector?

File: frontend/src/components/header/styledHeader.js [hunk 2]
```diff
@@ -58,16 +55,20 @@ export const NavLink = styled(Link)`
   font-weight: 700;
   color: #000;
   text-decoration: none;
-  transition: opacity 0.3s ease;
+  transition: color 0.15s ease;
   &:hover {
-    opacity: 0.7;
+    color: rgba(112, 0, 255, 1);
```
Comment (rileydanejohnston) [inline frontend/src/components/header/styledHeader.js:60]: rgba value over hex value? I see that they are equivalent I'm just curious why you went with rgba.

**Review by rileydanejohnston — APPROVED** [review]
Looks great!

File: frontend/src/components/header/styledHeader.js (hunk 2 above)
Comment (McCambley) [reply frontend/src/components/header/styledHeader.js:60]: Ahh yeah, just copied from the figma, I could probably convert it though huh?

File: frontend/src/components/header/styledHeader.js (hunk 1 above)
Comment (McCambley) [reply frontend/src/components/header/styledHeader.js:22]: Oh yep, good catch. Perhaps that can be cleaned up in the next PR o mine

**Pull request merged**